.travis.yml

```diff
@@ -37,19 +42,22 @@ before_install:
   - sudo mkdir $HOME
   - sudo chown "$USER:" $HOME
   - LINUXBREW=$HOME/.linuxbrew
-  - git clone https://github.com/Linuxbrew/brew $LINUXBREW
+  - git clone --depth=1 https://github.com/Linuxbrew/brew "$LINUXBREW"
```

Comment:
Core currently has ".git" at the end of the URL; while I had previously removed that from core (https://github.com/Linuxbrew/homebrew-core/pull/1790), both core and science currently appear to have it, so it might make sense to put it back in and reduce the diff size between repos.